user-storage-jpa/src/test/java/org/keycloak/quickstart/ArquillianJpaStorageTest.java

```diff
@@ -21,54 +21,64 @@
 import org.jboss.arquillian.drone.api.annotation.Drone;
 import org.jboss.arquillian.graphene.page.Page;
 import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.test.api.ArquillianResource;
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.keycloak.quickstart.page.ConsolePage;
 import org.keycloak.quickstart.storage.user.EjbExampleUserStorageProvider;
 import org.keycloak.quickstart.storage.user.EjbExampleUserStorageProviderFactory;
 import org.keycloak.quickstart.storage.user.UserAdapter;
 import org.keycloak.quickstart.storage.user.UserEntity;
+import org.keycloak.representations.idm.ComponentRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.keycloak.test.FluentTestsHelper;
 import org.keycloak.test.page.LoginPage;
-import org.openqa.selenium.By;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.online.ManagementProtocol;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineOptions;
 
-import java.io.IOException;
-import java.net.MalformedURLException;
-import java.net.URL;
+import javax.ws.rs.core.Response;
 import java.util.concurrent.TimeUnit;
 
 import static java.lang.String.format;
-import org.jboss.arquillian.graphene.Graphene;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
 
 @RunWith(Arquillian.class)
 public class ArquillianJpaStorageTest {
 
-    private static final String KEYCLOAK_URL = "http://%s:%s/auth%s";
 
-    @Page
-    private LoginPage loginPage;
+    public static final String KEYCLOAK_URL = "http://localhost:8180/auth";
```

Comment:
@vmuzikar I don't see any harm about getting these changes in as is. But would make sense to make this URL dynamic? I guess we used to do that before.

Reply:
Yes, some tests try to do that but it doesn't ever really fully work. AFAIK all the QS basically expect KC to be running at this address (not just the tests) so IMHO it is ok to hardcode it for testing purposes.